example/main.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""
+Documentation
+
+See also https://www.python-boilerplate.com/flask
+"""
+import os
+
+from flask import Flask, jsonify
+from flask_cors import CORS
+
+from featurehub_sdk.featurehub_config import FeatureHubConfig
+
+
+def create_app(config=None):
+    app = Flask(__name__)
+
+    # See http://flask.pocoo.org/docs/latest/config/
+    app.config.update(dict(DEBUG=True))
+    app.config.update(config or {})
+
+    # Setup cors headers to allow all domains
+    # https://flask-cors.readthedocs.io/en/latest/
+    CORS(app)
+
+    # Definition of the routes. Put them into their own file. See also
+    # Flask Blueprints: http://flask.pocoo.org/docs/latest/blueprints
+
+
+    # Create FeatureHub configuration
+    edge_url = 'https://zjbisc.demo.featurehub.io/'
+    client_eval_key = 'default/806d0fe8-2842-4d17-9e1f-1c33eedc5f31/wFk8qfmJEkLnkkJ8A8ZPTbgZPHPTJJ*heGuAGL6U8EKOUXbbRCL'
+
+    config = FeatureHubConfig(edge_url, client_eval_key)
+    config.init()
+    fh = config.repository()
```

Comment:
what is "fh" for?

Reply:
does this app not start until you have contacted the feature repository?

Reply:
yes I think so, is it not right?